### ISSUE TITLE
Support for the HTTP requestType is missing

### ISSUE THREAD

**Issue by JackLivio** [smartdevicelink/sdl_android#292]
HTTP RequestType support is missing - this is required for unencrypted policy table updates to work in SDL Core.

Currently when an OnSystemRequest is received, if the request type is not proprietary, the proxy will not send the message to the server. [If statement here](https://github.com/smartdevicelink/sdl_android/blob/master/sdl_android_lib/src/com/smartdevicelink/proxy/SdlProxyBase.java#L2874).

In addition to providing  support for an HTTP onSystemRequest request_type, when receiving this type of message, the proxy also needs to construct the httprequest header and add it to the incoming message before sending to the server. 

Here is what the httprequest should look like
```
{
            "HTTPRequest": {
                        "headers": {
                                   "ContentType": "application/json",
                                   "ConnectTimeout": <int value from policy table:timeout_after_x_seconds>,
                                   "DoOutput": true,
                                   "DoInput": true,
                                   "UseCaches": false,
                                   "RequestMethod": "POST",
                                   "ReadTimeout": <int value from policy table:timeout_after_x_seconds>,
                                   "InstanceFollowRedirects": false,
                                   "charset": "utf-8",
                                   "Content_Length": <value of the length of the "body" string>
                        },
                        "body": "{"data":["this is either the syncp packet or raw policy table BgcYTArvdz62FO7yHgOTXtE3EP0E0WsbFeDqzYY="]}"
            }
}
```

**Comment by JackLivio** [comment]
Working off of [this branch](https://github.com/smartdevicelink/sdl_android/tree/hotfix/handle_http_on_system_requests)

**Comment by joeygrover** [comment]
Fixed wth #293
